files/en-us/web/api/webxr_device_api/geometry/index.html

```diff
@@ -83,7 +83,7 @@ <h2 id="On_the_origins_of_spaces">On the origins of spaces</h2>
 
 <p>In WebXR, the fundamental concept of a <strong>space</strong>—as in, a coordinate space in which a scene takes place—is represented by an instance of {{domxref("XRSpace")}}. The space is used to make determinations about the relative positions and motion of objects and other entities (such as light sources and cameras) within the user's environment.</p>
 
-<p>A point in 3D space is, as covered previously, made up of three components, each identifying the distance of the point along one the three axes. The position of any given point in the world is described by indicating how far along each of three axes the point is located from the center of the space, which is the position at which those the three axes meet.</p>
+<p>As mentioned earlier, any given 3D point consists of three components each identifying the distance from the center of the space along one of the three axes.</p>
```

Comment:
```suggestion
<p>As mentioned earlier, any given 3D point consists of three components, each identifying the distance from the center of the space along one of the three axes.</p>
```
The comma seems necessary here